Planet.pde

```diff
@@ -10,14 +10,16 @@ class Planet {
   color col;
 
   boolean onScreen = true;
-
+  boolean explode_me;
```

Comment:
Please rename this variable to `explodeMe`, to comply with the naming style of the rest of the variables.

Please also move the ` = false` out of the constructor into the raw class